data/json/recipes/recipe_others.json

```diff
@@ -326,6 +326,34 @@
     ]
   },{
   "type" : "recipe",
+  "result": "tailors_kit",
+  "category": "CC_OTHER",
+  "subcategory": "CSC_OTHER_TOOLS",
+  "skill_used": "tailor",
+  "difficulty": 3,
+  "time": 15000,
+  "reversible": true,
+  "autolearn": false,
+  "book_learn": [[ "textbook_tailor", 3 ] , [ "manual_tailor", 3 ]],
+  "components": [
+    [
+      [ "sewing_kit", 1 ]
+    ],
+    [
+      [ "scissors", 1 ]
+    ],
+    [
+      [ "knitting_needles", 1 ]
+    ],
+    [
+      [ "lighter", 1 ]
+    ],
+    [
+      [ "plastic_chunk", 3 ]
```

Comment:
OK, if these are to expand the amount of room in the kit, might need a soldering iron for the reshape.  Easy enough to add in the merge.